src/array/take.js

```diff
@@ -0,0 +1,17 @@
+import _take from 'lodash/fp/take'
```

Comment:
`take` and its variants don't mutate their args, you should import it from `'lodash/take'` instead of `'lodash/fp/take'`.

Then you should be able to use `convert` instead of `convertLodashFp`.

The problem you were reporting to me when using `convert` was coming from the fact that you were importing `take` from `'lodash/fp/take'` (rearged and curried version).

The same goes for the three other functions.